src/async-wrap-inl.h

```diff
@@ -47,7 +45,7 @@ inline AsyncWrap::AsyncWrap(Environment* env,
   if (try_catch.HasCaught())
     FatalError("node::AsyncWrap::AsyncWrap", "init hook threw");
 
-  has_async_queue_ = true;
+  bits_ |= 1;  // has_async_queue() is true now.
```

Comment:
I agree it's not super pretty.  Once I find a few more spots like this, I'll introduce a BitField class.